host-controller/src/main/java/org/jboss/as/host/controller/logging/HostControllerLogger.java

```diff
@@ -1265,4 +1266,12 @@ void noDomainControllerConfigurationProvidedForAdminOnly(String policyAttribute,
     @LogMessage(level = Level.ERROR)
     @Message( id = 172, value = "Failed executing the suspend operation for server: %s")
     void suspendExecutionFailed(@Cause IOException cause, String serverName);
+
+    @LogMessage(level = Level.INFO)
```

Comment:
I agree that if any servers failed to suspend that details of that should be provided to the caller (i.e. the OperationStepHandlers that call this method) and those OSHs should use that data to compose a failure message and pass it to OperationContext.getFailureDescription(). The user should see the failure.